modules/pgp/src/test/resources/file-decryption-config-no-secret-key.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:spring="http://www.springframework.org/schema/beans" xmlns:pgp="http://www.mulesoft.org/schema/mule/pgp"
+	xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+          http://www.mulesoft.org/schema/mule/pgp http://www.mulesoft.org/schema/mule/pgp/current/mule-pgp.xsd
+          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd
+          http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd">
+
+	<spring:bean id="pgpKeyManager" class="org.mule.module.pgp.PGPKeyRingImpl"
+		init-method="initialise">
+		<spring:property name="publicKeyRingFileName" value="./serverPublic.gpg" />
+		<spring:property name="secretAliasId" value="6247672658342245276" />
+		<spring:property name="secretPassphrase" value="TestingPassphrase" />
+	</spring:bean>
+
+	<spring:bean id="fakeCredentialAccessor"
+		class="org.mule.module.pgp.FakeCredentialAccessor">
+		<spring:property name="credentials"
+			value="Mule server &lt;mule_server@mule.com&gt;" />
+	</spring:bean>
+
+	<pgp:security-manager>
+		<pgp:security-provider name="pgpSecurityProvider"
+			keyManager-ref="pgpKeyManager" />
+		<pgp:keybased-encryption-strategy
+			name="keyBasedEncryptionStrategy" credentialsAccessor-ref="fakeCredentialAccessor"
+			keyManager-ref="pgpKeyManager" />
+	</pgp:security-manager>
+
+	<flow name="pgpDecryptProcessor">
+
+		<vm:inbound-endpoint path="inDecrypt" />
+
+		<vm:outbound-endpoint path="outDecrypt">
```

Comment:
Can you avoid using vm outbound endpoint by using the returned payload? Maybe even in the custom processor.